src/decoder.rs

```diff
@@ -43,9 +43,9 @@ quick_error! {
             display("Invalid Chunk header: {err:x?}")
         }
 
-        /// Some bits were invalid
-        ReservedBitSet {
```

Comment:
Removing this would be an API breaking change. Not sure whether `quick_error!` supports marking variants as deprecated, but if so, I think that would be the thing to do

Reply:
that makes a lot of sense actually, fixed on most recent commit!